src/main/java/com/palantir/docker/compose/connection/ContainerAccessor.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.docker.compose.connection;
+
+public interface ContainerAccessor {
```

Comment:
I almost did this... using a longer method than just `get` means that both the `ContainerCache` and the underlying `DockerCompose` actually conform to this interface, which seemed oddly pleasing.

What about calling it `Containers` but keeping the `container(String name)` method?